src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/hearings/hearingdetails/HearingDetailServiceTest.java

```diff
@@ -91,6 +91,45 @@ public void testHandleListingSelected() {
         assertEquals(notes, caseData.getHearingDetailsHearingNotes2());
     }
 
+    @Test
+    public void testHandleListingSelectedNullValue() {
+        CaseData caseData = createCaseData();
+        selectedListing.setHearingStatus(null);
+        selectedListing.setPostponedBy(null);
+        selectedListing.setHearingCaseDisposed(null);
+        selectedListing.setHearingPartHeard(null);
+        selectedListing.setHearingReservedJudgement(null);
+        selectedListing.setAttendeeClaimant(null);
+        selectedListing.setAttendeeNonAttendees(null);
+        selectedListing.setAttendeeRespNoRep(null);
+        selectedListing.setAttendeeRespAndRep(null);
+        selectedListing.setAttendeeRepOnly(null);
+        selectedListing.setHearingTimingStart(null);
+        selectedListing.setHearingTimingBreak(null);
+        selectedListing.setHearingTimingResume(null);
+        selectedListing.setHearingTimingFinish(null);
+        selectedListing.setHearingTimingDuration(null);
+        selectedListing.setHearingNotes2(null);
+        hearingDetailsService.handleListingSelected(caseData);
+
+        assertEquals(" ", caseData.getHearingDetailsStatus());
+        assertEquals(" ", caseData.getHearingDetailsPostponedBy());
+        assertEquals(" ", caseData.getHearingDetailsCaseDisposed());
+        assertEquals(" ", caseData.getHearingDetailsPartHeard());
+        assertEquals(" ", caseData.getHearingDetailsReservedJudgment());
+        assertEquals(" ", caseData.getHearingDetailsAttendeeClaimant());
+        assertEquals(" ", caseData.getHearingDetailsAttendeeNonAttendees());
+        assertEquals(" ", caseData.getHearingDetailsAttendeeRespNoRep());
+        assertEquals(" ", caseData.getHearingDetailsAttendeeRespAndRep());
+        assertEquals(" ", caseData.getHearingDetailsAttendeeRepOnly());
+        assertEquals(" ", caseData.getHearingDetailsTimingStart());
+        assertEquals(" ", caseData.getHearingDetailsTimingBreak());
+        assertEquals(" ", caseData.getHearingDetailsTimingResume());
+        assertEquals(" ", caseData.getHearingDetailsTimingFinish());
+        assertEquals(" ", caseData.getHearingDetailsTimingDuration());
+        assertEquals(" ", caseData.getHearingDetailsHearingNotes2());
```

Comment:
Minor: prefer Hamcrest style of asserts...
`assertThat(something).isEqualTo(somethingElse);`
As this is slightly more readable